### PR TITLE
app-backup/snapper: Bump to 0.5.4

### PR DESCRIPTION
**Pull request by jubalh**
Package-Manager: Portage-2.3.19, Repoman-2.3.6

### CODE REVIEW

File: app-backup/snapper/snapper-0.5.4.ebuild [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+	sys-devel/gettext
+	virtual/pkgconfig"
+
+REQUIRED_USE="|| ( btrfs ext4 lvm )"
+
+DOCS=( AUTHORS )
+
+src_prepare() {
+	default
+	epatch "${FILESDIR}"/cron-confd.patch
```
Comment (a17r) [inline app-backup/snapper/snapper-0.5.4.ebuild]: `EAPI 6`, so just drop src_prepare and put the patch into `PATCHES` array.

File: app-backup/snapper/snapper-0.5.4.ebuild [hunk 2]
```diff
@@ -0,0 +1,73 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit eutils
```
Comment (a17r) [inline app-backup/snapper/snapper-0.5.4.ebuild]: only `prune_libtool_files` is used so you can inherit `ltprune` instead. But if you do `find "${D}" -name '*.la' -delete || die` instead you can drop inherit completely.

File: app-backup/snapper/snapper-0.5.4.ebuild [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+	pam? ( sys-libs/pam )
+	xattr? ( sys-apps/attr )"
+
+DEPEND="${RDEPEND}
+	sys-devel/gettext
+	virtual/pkgconfig"
+
+REQUIRED_USE="|| ( btrfs ext4 lvm )"
+
+DOCS=( AUTHORS )
```
Comment (a17r) [inline app-backup/snapper/snapper-0.5.4.ebuild]: you can probably drop that, it will be picked automatically

**Comment by jubalh** [comment]
Thanks for the review :)

**Pull request closed without merging**